documents/motions/15.2022.1 - Suspensions and other Sanctions.md

```diff
@@ -1,45 +1,54 @@
+2. The WCA Integrity Committee may exercise its suspensive rights under this Motion only if:
    1. a Registered Speedcuber has failed to pay fees for WCA Competitions, or
-   2. in the opinion of the WDC, the Registered Speedcuber is in breach of the Bylaws, Motions, or Regulations, or
-   3. the conduct of the Registered Speedcuber breaches or remains in breach of the Mission, Spirit, and/or Objectives of the WCA, or
-   4. the WDC considers that the Registered Speedcuber does not fulfil the requirements of eligibility for registration as a Registered Speedcuber.
-3. Before the WDC may exercise its suspensive rights under this Motion, the Registered Speedcuber must have been sent notice of the grounds and must have been given the opportunity to be heard by a WCA Delegate, the WCA Disciplinary Committee, or the WCA Board.
-4. A Registered Speedcuber who has been suspended by the WDC under this Motion may request for reinstatement by the WDC, with a motivation of the request.
-5. A Registered Speedcuber who has been suspended by the WDC under this Motion shall automatically cease to be suspended at the end of the fixed period, or once, in the opinion of the WDC, the set of circumstances specified has changed or ceases to exist.
-6. The WCA Ethics Committee shall have the following rights and duties, based on a Voting in the WCA Ethics Committee with a supermajority:
-   1. to issue fines against a Registered Speedcuber and suspend a Registered Speedcuber from participation in a WCA Competition until the fines are paid.
-      1. Fines must only be issued if acts of the Registered Speedcuber caused financial damage or financial loss to the WCA or any organization or individual within the WCA Community.
+   2. in the opinion of the WCA Integrity Committee, the Community member is in breach of the Spirit, Mission, Motions, Policies, Codes and/or Regulations of the WCA, or
+   3. the WCA Integrity Committee considers that the Registered Speedcuber does not fulfil the requirements of eligibility for registration as a Registered Speedcuber, or
+   4. the WCA Integrity Committee considers that the presence of a the Community member is a threat to the safety of other Community members or otherwise disruptive for the community.
+3. Before the WCA Integrity Committee may exercise its suspensive rights under this Motion, the Community member must have been sent notice of the grounds and must have been given the opportunity to provide a statement to the WCA Integrity Committee.
+4. A Community member who has been suspended by the WCA Integrity Committee under this Motion may request for reinstatement by the WCA Appeals Committee, with a motivation of the request.
+5. A Community member who has been suspended by the WCA Integrity Committee under this Motion shall automatically cease to be suspended at the end of the fixed period, or once, in the opinion of the WDC, the set of circumstances specified has changed or ceases to exist.
```

Comment:
lingering WDC reference, also goes back to the infinite appeals point we just discussed

Reply:
WDC reference removed however the line kept in as this allows the WIC to unban someone who for example has been banned until an apology is written or until stolen items are returned